assistant/cyton/cyton_controller.py

```diff
@@ -65,15 +68,71 @@ def set_pose(self, q: SE3):
                 self.sock.send(q)
         # TODO: fake printouts
 
-    def go_home(self):
-        self.set_pose(self.robot.qz)
 
-    def grab_object(self, item: BaseItem):
-        robot_object_pose = self.robot.ikine_LM(item.pose, q0=self.pose)
+    def open_gripper(self):
+        """
+        Opens the gripper
+        :return: None
+        """
+        self.robot.open_gripper()
```

Comment:
Similar to the other comment, lets move the logic for moving the gripper into a dummy function like

```python
def set_gripper(self, gripper_value):
    #dosomething
    pass
```